### PR TITLE
add input and chat css; resolve merge conflicts; update overall css

### PR DESCRIPTION
**Pull request by timmy224**
### **Permission status moved to top of app**
Screen Shot 2020-11-11 at 8 26 25 AM

### **Add input message CSS, resolved merge conflicts, fixed CSS for sidebar, logout button, padding issues, etc.**
Screen Shot 2020-11-11 at 8 26 29 AM

@BitPhoenix @luiszaval @Sleyter2616 - mentioning all to make sure we all agree with CSS changes made before merging into week-6 branch

### CODE REVIEW

File: src/components/Chat/Chat.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#box-first{
+    height: 10vh;
+}
+
+#box-fill {
+    display: flex;
+    height: 80vh;
+    flex: 1;
+    width: 100%;
+}
```
Comment (BitPhoenix) [inline src/components/Chat/Chat.css:23]: Nice job spacing out the styles!

File: src/components/MainComponent/MainComponent.js [hunk 2]
```diff
@@ -44,42 +45,35 @@ class MainComponent extends Component {
...
+                                    resource="channel-member"
+                                    action="add"
+                                    yes={() => <p>User can add channel members</p>}
+                                    no={() => <p>User cannot add channel members</p>}
+                                />
+                            </div>
                         </div>
-                        <div className="row">
-                            <div className="col-3">
+                        <div className="col-12 row sidebar-chat-wrapper">
```
Comment (BitPhoenix) [inline src/components/MainComponent/MainComponent.js:69]: I think we might be able to get rid of the col-12 in front and just have

`row sidebar-chat-wrapper`

File: src/components/SideBar/Sidebar.css [hunk 3]
```diff
@@ -32,15 +40,14 @@
     flex: 1 1 0;
     display: block;
     text-align: left;
-    padding-right: 25px;
+    /* padding-right: 25px; */
```
Comment (BitPhoenix) [inline src/components/SideBar/Sidebar.css:43]: Comment remove

File: src/components/SideBar/Sidebar.css [hunk 4]
```diff
@@ -32,15 +40,14 @@
...
 }
 
 .sidebar-section-heading-right {
     display: flex;
     align-items: center;
     margin-left: auto;
-    margin-left: 8px;
-    padding: 0 8px;
+    /* padding: 0 8px; */
```
Comment (BitPhoenix) [inline src/components/SideBar/Sidebar.css:50]: Comment remove

File: src/components/SideBar/Sidebar.css [hunk 5]
```diff
@@ -70,4 +77,26 @@
...
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: flex-end;
+}
+
+.logout-btn {
+    /* justify-content: center;
+    display: flex;
+    align-items: flex-end; */
```
Comment (BitPhoenix) [inline src/components/SideBar/Sidebar.css:98]: Comment remove

**Review by BitPhoenix — CHANGES_REQUESTED** [review]

**Comment by BitPhoenix** [comment]
@timmy224 
Getting strange layout when having multiple messages 

image

**Comment by timmy224** [comment]
Updated CSS for messages display portion of chat Component.
Screen Shot 2020-11-11 at 11 11 15 AM

File: src/components/MainComponent/MainComponent.js (hunk 2 above)
Comment (timmy224) [reply src/components/MainComponent/MainComponent.js]: done

**Pull request merged**